records_mover/airflow/hooks/records_hook.py

```diff
@@ -41,7 +41,8 @@ def _get_boto3_session(self) -> boto3.session.Session:
     def _url_resolver(self) -> UrlResolver:
         return UrlResolver(boto3_session_getter=self._get_boto3_session,
                            gcs_client_getter=lambda: None,
-                           gcp_credentials_getter=lambda: None)
+                           gcp_credentials_getter=lambda: None,
+                           gcp_project_id=None)
```

Comment:
GCP Data Transfer Service requires the project ID associated with the GCS bucket, so let's keep track of that.